cmd/version.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+
+	goVersion "github.com/christopherhein/go-version"
```

Comment:
So looking over Istio's version command, one of the things I like is that it prints both git commit and "version", where version is populated by the tag (if the commit has a tag; if not the version is something like `unknown` or `<git commit sha>-clean`/`<git commit sha>-dirty`). It's easy enough to add, but won't work with this library.

Then looking at the library, I don't think it's worth vendoring period. The library itself is fine, but one of the go-ism is ["a little copying is better than a little dependency"](https://go-proverbs.github.io/) (also auditing the code I don't love how `ToShortened` shortened is implemented, if I'm nitpicking things :P ). Given that the dependency is so simple, a struct that's just a wrapper around `json.Marshal` that also implements `ToString`, I'd rather just see us implement it ourselves in this package. Then we can easily include a version/tag field, for example.

Reply:
Will do, good practice for me